src/System.Drawing.Common/src/System/Drawing/Printing/LibcupsNative.cs

```diff
@@ -9,10 +9,14 @@ namespace System.Drawing.Printing
 {
     internal static class LibcupsNative
     {
-        private const string LibraryName = "libcups";
-        private static IntPtr s_libcupsHandle = LoadLibcups();
+        internal const string LibraryName = "libcups";
 
-        private static IntPtr LoadLibcups()
+        static LibcupsNative()
+        {
+            LibraryResolver.EnsureRegistered();
```

Comment:
Do we need to call this also in `SafeNativeMethods.Gdip` and `UnsafeNativeMethods.Gdip` ?

Reply:
Yeah, I feel particularly stupid now, I was trying to build the Linux bits and mistyped the property name so I was getting the Windows build and didn't notice. Derp.